docs/mcd.md

```diff
@@ -0,0 +1,244 @@
+# Tableau Connector
+
+**IMPORTANT:** This is alpha software and should be used in a test environment. Do not deploy the connector to a production environment.
+
+Tableau Connector is a new feature that enables a more fully data-driven connection dialog for plugin connectors. Additionally, it provides some control over the metadata hierarchy elements--Database, Schema, and Table--both in the connection dialog and in the schema viewer, which the user sees after the connection is established.
+
+This feature is new in 2020.2. It is an alpha feature, and as such it is available in Desktop only. In future releases it will become available in other Tableau products including Server and Prep. 
+
+# How to Use Tableau Connector
+
+To use Tableau Connector, in the manifest replace `<connection-dialog>`, which references a .tcd file, with `<connection-fields>`, which references a .xml file, described below. The connector will fail to load if both elements appear in the manifest.
```

Comment:
This is probably nitpicky, but a .tcd file is an xml file. Plus, I think we're moving away from using the file extensions because of the confusion (for example, .tcd and .tdc are easily confused). You can remove the references to file extensions here to avoid confusion.

Reply:
Done. Fewer words is better as long as it's understandable. :)